docs/release-notes.rst

```diff
@@ -6,6 +6,98 @@
  Release Notes
 ###############
 
+**************
+ Version 0.34
+**************
+
+Version 0.34.0
+==============
+
+**Release Date:** June 24, 2024
+
+**Breaking Changes**
+
+-  Images: The default environment includes images that support PyTorch. TensorFlow users must
+   configure their experiments to target our non-default TensorFlow images. Details on this process
+   can be found at :ref:`set-environment-images`
```

Comment:
-  Images: The default environment includes images that support PyTorch. Therefore, TensorFlow users must
   configure their experiments to target our non-default TensorFlow images. Details on this process
   can be found at :ref:`set-environment-images`